backend/app.py

```
@@ -0,0 +1,203 @@
+import os
+import json
+
+from models import UserLogin, UserRegister, ItemCreate, SearchQuery, ChatMessage, AuthResponse, ItemResponse, ChatResponse
+from auth import login_user, register_user
+from databases.sql import init_db, create_item as db_create_item, search_items as db_search_items, get_user_items as db_get_user_items
+from databases.qdrant import init_qdrant, store_item_vector, search_similar_items
+from utils import process_item_data, chat_with_database, generate_embedding
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.16.0)

[warning] Unused function argument: `app`

(ARG001)